ang/crmMailing/services.js

```diff
@@ -156,7 +156,7 @@
             mailings: {include: [], exclude: []}
           },
           template_type: "traditional",
-          template_options: "",
+          template_options: {nonce: 1}, // Workaround CRM-19756
```

Comment:
OK, I could put it on the preceding line. I like having the comment adjacent to the code.